.github/labeler.yml

```
@@ -6,15 +6,15 @@
 - changed-files:
   - any-glob-to-any-file: ['.github/*', '**/*.yml', '.gitattributes', '.gitignore']
 
-"JSON":
+"Area: JSON":
 - changed-files:
   - all-globs-to-any-file: ['**/*.json']
 
-"Audio":
+"Area: Audio":
 - changed-files:
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)